modules/gdscript/editor/script_templates/CharacterBody2D/godot_character_controller.gd

```diff
@@ -0,0 +1,45 @@
+# meta-description: Godot character controller for gravity games (platformer, ...)
+
+extends _BASE_
+
+
+@export_group("Kinematic")
+@export var walk_max_speed := 500.0
+@export var walk_acceleration := 1500.0
+@export var run_max_speed := walk_max_speed * 1.5
+@export var run_acceleration := walk_acceleration * 1.5
+@export var friction := 1000.0
+@export var slide_friction := 1000.0
+@export var jump_velocity := -400.0
+
+# Get the global gravity from the project settings. RigidBody2D nodes also use this value.
+var gravity: float = ProjectSettings.get_setting("physics/2d/default_gravity")
+
+
+func _physics_process(delta: float) -> void:
+	# The run action is not defined by default in the project, you must add it yourself.
+	var running = Input.is_action_pressed("run")
```

Comment:
All the other inputs use the default input actions, this one probably should too. Otherwise we should require all custom ones and add some comments about needing to add them.